libpius/signer.py

```diff
@@ -645,6 +648,9 @@ def sign_uid(self, key, index, level):
     while True:
       line = gpg.stdout.readline()
       debug('Got %s' % line)
+      if 'MISSING_PASSPHRASE' in line:
```

Comment:
Just so that my cleanup is easier later can you add a comment here that this is gpg2 specific, where as BADD_PASSHRASE is gpg1?